pkg/webhook/server.go

```diff
@@ -526,9 +526,7 @@ func injectFluentBit(pod *corev1.Pod) (bool, error) {
 			if name := volumes[i].Name; name == value {
 				sidecar.VolumeMounts = append(sidecar.VolumeMounts, corev1.VolumeMount{
 					Name:      name,
-					MountPath: "/fluent-bit/etc/fluent-bit.conf",
-					SubPath:   "fluent-bit.conf",
-				})
+					MountPath: "/fluent-bit/etc" })
```

Comment:
I think that the behavior about custom config volume should be same as fluentd, so could you please rewrite to mount directory like fluent-bit?

Reply:
Ok fair enough. See commit ca7630d for it.